### PR TITLE
Series plugin blog post

### PR DESCRIPTION
**Pull request by jtama**

### CODE REVIEW

File: blog/content/posts/2024-12-06-series.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: :theme/post
+title: Want to a blog post series ?
```
Comment (ia3andy) [inline blog/content/posts/2024-12-06-series.adoc]:
```suggestion
title: Do you want to publish a blog post series ?
```

Comment (jtama) [reply blog/content/posts/2024-12-06-series.adoc]: Obviously forgot a word...

Wanted to write : _Want to write a blog post series ?_

But will keep you suggestion.

**Comment by jtama** [comment]
Also but that's an other point, I think we should enable `futur` for pr builds.

**Comment by jtama** [comment]
> Yes! go for it!!!

Done.

**Comment by ia3andy** [comment]
Thanks @jtama !!

**Pull request merged**